metadata.json

```diff
@@ -76,10 +75,6 @@
   ],
   "requirements": [
     {
-      "name": "pe",
-      "version_requirement": "3.x"
-    },
-    {
       "name": "puppet",
       "version_requirement": ">= 2.7.0 < 5.0.0"
```

Comment:
If we leave it puppet >= 2.7.0 then we have to do a major bump to DROP 2.7 and 3.x, which we are sad to have to do for other modules.